aesara/scalar/basic.py

```diff
@@ -793,8 +793,8 @@ def __sub__(self, other):
     def __mul__(self, other):
         return mul(self, other)
 
-    def __truediv__(self, other):
-        return true_div(self, other)
+    def __truedivide__(self, other):
```

Comment:
Is `__truedivide__` an actual Python Dundee method?

Reply:
Yes, I checked again and found [`__truediv__`](https://docs.python.org/3/library/operator.html#operator.__truediv__) is a python operator so I have a correct this, Thanks for correcting me.

Reply:
Ha, I like that spell check on my phone changed "dunder" to "Dundee"!

Reply:
Seems like we need some kind of meme GIF for [this situation](https://knowyourmeme.com/memes/thats-a-knife).

Reply:
ha ha ha 🤣